cardano-cli/test/cardano-cli-golden/files/golden/governance/stakeaddress/poolAndNoConfidenceDeleg.cert

```diff
@@ -1,5 +1,5 @@
 {
-    "type": "CertificateConway",
+    "type": "Certificate",
```

Comment:
We should make sure that `cardano-testnet` and `cardano-node-tests` pass with this change. I guess the change to `legacyComparison` here https://github.com/IntersectMBO/cardano-api/pull/962/files should do the job.